integration/kubernetes/init.sh

```diff
@@ -117,6 +117,13 @@ sudo systemctl status "${cri_runtime}" --no-pager
 if [ "${stress_image_pull_policy}" == "Always" ]; then
 	echo "Store custom stress image in registry"
 	sudo -E "${container_engine}" run -d -p ${registry_port}:5000 --name "${registry_name}" "${registry_image}"
+	# wait for registry container
+	for i in $(seq 1 5); do
```

Comment:
You could use our https://github.com/kata-containers/tests/blob/4cd975e0067a1c3ff59829d8d5c476c2f4e7e60d/.ci/lib.sh#L173 here